extensions/sql-migration/src/wizard/loginMigrationTargetSelectionPage.ts

```diff
@@ -344,12 +345,33 @@ export class LoginMigrationTargetSelectionPage extends MigrationWizardPage {
 			await this.populateLocationDropdown();
 
 			// Collect source login info here, as it will speed up loading the next page
-			const sourceLogins: LoginTableInfo[] = [];
+			var sourceLogins: LoginTableInfo[] = [];
 			sourceLogins.push(...await collectSourceLogins(
 				await getSourceConnectionId(),
 				this.migrationStateModel.isWindowsAuthMigrationSupported));
+
+			// validate Login Eligibility result contains system logins in Exception map from which system login names can be extracted.
+			// These system logins are not to be displayed in the source logins list
+			var validateLoginEligibilityResult: contracts.StartLoginMigrationPreValidationResult | undefined = await this.migrationStateModel.migrationService.validateLoginEligibility(
+				await getSourceConnectionString(),
+				"",
+				sourceLogins.map(row => row.loginName),
+				""
+			);
+
+			var sourceSystemLoginsName: string[] = [];
+
+			if (validateLoginEligibilityResult !== undefined) {
```

Comment:
do we need to handle when it is undefined ? may be log some error.

Reply:
done